.github/actions/upload-build-artifacts/action.yml

```diff
@@ -0,0 +1,41 @@
+name: "Upload build artifacts"
+description: "Upload build artifacts"
+
+runs:
+  using: "composite"
+  steps:
+    ## NXCACHE
+    - name: tar artifacts
+      run: |
+        tar czf dist.tar.gz dist
+        tar czf nxcache.tar.gz nxcache
+        tar czf adf-cli.tar.gz node_modules/@alfresco/adf-cli
+        tar czf adf-testing.tar.gz node_modules/@alfresco/adf-testing
+      shell: bash
+    - name: update NX cache folder
+      uses: actions/upload-artifact@v3
+      continue-on-error: true
```

Comment:
are those necessary?

Reply:
yep, nx cache needs to be updated before launching e2e tests in order to use the nx affected command